flag_engine/identities/models.py

```diff
@@ -18,6 +19,7 @@ class IdentityModel:
     created_date: datetime = field(default_factory=datetime.datetime.now)
     identity_features: typing.List[FeatureStateModel] = field(default_factory=list)
     identity_traits: typing.List[TraitModel] = field(default_factory=list)
+    identity_uuid: str = field(default_factory=uuid.uuid4)
```

Comment:
I think this should just be `uuid`?

Reply:
flake8 cries about that since uuid is also the name of the package.

Reply:
Ok, fair enough.